### ISSUE TITLE
Fix S2629 FP: Constant interpolated string concatenation

### ISSUE THREAD

**Issue by Jake-Derrick** [SonarSource/sonar-dotnet#9653]
### Description

Please provide a succinct description of your issue. Also please provide the rule ID (e.g. S1234)

### Repro steps

```csharp
const string localConstant = "local";

logger.Log(LogLevel.Warning, $"Constant: {localConstant}" + $"Constant2: {localConstant}"); // Non-Compliant FP

logger.Log(LogLevel.Warning, $"Constant: {localConstant} Constant2: {localConstant}"); // Compliant

```
### Expected behavior

Rule S2629 not raised

### Actual behavior

Rule S2629 raised

### Known workarounds

Use a single interpolated string

### Related information

* C#/VB.NET Plugins version - `9.27.0.93347`
* Visual Studio version - `17.10.1`
* MSBuild / dotnet version - `MSBuild 6.2`

**Comment by sebastien-marichal** [comment]
Hello @Jake-Derrick,

Thank you for reporting this issue.

I confirm this is a false positive.
